modules/planning/README.md

```diff
@@ -16,15 +16,19 @@ use cases as they were all treated as a single driving scenario.
 Apollo 5.5, which focuses on curb-to-curb autonomous driving on urban roads,
 introduced 2 new planning scenarios.
 
-Apollo 6.0, which focuses on intellegence of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode. 
+Apollo 6.0, which focuses on intellegence and data-driven development of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode, by which we demonstrate the new capability of dynamically nudginng a moving obstacle. These two modes also define a series of APIs where developers can generate thier own training data and brought in their own models.
```

Comment:
`intellegent` as an adj for `development` ? 
`thier` should be `their`. `brought in` should be `bring in`.